src/index.jsx

```diff
@@ -37,6 +38,7 @@ export function Auth0(props) {
     clientID: auth0config.clientId,
     audience: auth0config.audience,
     redirectUri: auth0config.redirectUri,
+    logoutUrl: auth0config.logoutUrl,
```

Comment:
could we use `${process.env.VITE_BASE_URL}/auth/logout` as a default here to make the prop optional?